module1/projects/connect_four/iteration_2.md

```diff
@@ -10,24 +10,148 @@ _[Back to Requirements](./requirements)_
+pry(main)> g_cells = [Cell.new("G1"),Cell.new("G2"),Cell.new("G3"),Cell.new("G4"),Cell.new("G5"),Cell.new("G6")]
+pry(main)> columns = {A: a_cells, B: b_cells, C: c_cells, D: d_cells, E: e_cells, F: f_cells, G: g_cells}
+pry(main)> board = Board.new(columns)
+pry(main)> c3 = board.columns[:C][2]
+pry(main)> e2 = board.columns[:E][1]
+pry(main)> board.row_for(c3)
+# => [#<Cell:0x0000000130333cc8 @column="A", @filled=false, @placement="A3", @row=3, @value=".">,
+ #<Cell:0x0000000130333bd8 @column="B", @filled=false, @placement="B3", @row=3, @value=".">,
+ #<Cell:0x0000000130333ae8 @column="C", @filled=false, @placement="C3", @row=3, @value=".">,
+ #<Cell:0x00000001303339f8 @column="D", @filled=false, @placement="D3", @row=3, @value=".">,
+ #<Cell:0x0000000130333908 @column="E", @filled=false, @placement="E3", @row=3, @value=".">,
+ #<Cell:0x0000000130333818 @column="F", @filled=false, @placement="F3", @row=3, @value=".">,
+ #<Cell:0x0000000130333728 @column="G", @filled=false, @placement="G3", @row=3, @value=".">]
+pry(main)> board.row_for(e2)
+# => [#<Cell:0x0000000130228e50 @column="A", @filled=false, @placement="A2", @row=2, @value=".">,
+ #<Cell:0x0000000130228b08 @column="B", @filled=false, @placement="B2", @row=2, @value=".">,
+ #<Cell:0x0000000130228518 @column="C", @filled=false, @placement="C2", @row=2, @value=".">,
+ #<Cell:0x0000000130228248 @column="D", @filled=false, @placement="D2", @row=2, @value=".">,
+ #<Cell:0x0000000130333f98 @column="E", @filled=false, @placement="E2", @row=2, @value=".">,
+ #<Cell:0x0000000130333ea8 @column="F", @filled=false, @placement="F2", @row=2, @value=".">,
+ #<Cell:0x0000000130333db8 @column="G", @filled=false, @placement="G2", @row=2, @value=".">]
+pry(main)> board.column_for(c3)
+# => [#<Cell:0x000000013022b100 @column="C", @filled=false, @placement="C1", @row=1, @value=".">,
+ #<Cell:0x0000000130228518 @column="C", @filled=false, @placement="C2", @row=2, @value=".">,
+ #<Cell:0x0000000130333ae8 @column="C", @filled=false, @placement="C3", @row=3, @value=".">,
+ #<Cell:0x0000000130333458 @column="C", @filled=false, @placement="C4", @row=4, @value=".">,
+ #<Cell:0x0000000130332dc8 @column="C", @filled=false, @placement="C5", @row=5, @value=".">,
+ #<Cell:0x0000000130332738 @column="C", @filled=false, @placement="C6", @row=6, @value=".">]
+pry(main)> board.column_for(e2)
+# => [#<Cell:0x000000013022a520 @column="E", @filled=false, @placement="E1", @row=1, @value=".">,
+ #<Cell:0x0000000130333f98 @column="E", @filled=false, @placement="E2", @row=2, @value=".">,
+ #<Cell:0x0000000130333908 @column="E", @filled=false, @placement="E3", @row=3, @value=".">,
+ #<Cell:0x0000000130333278 @column="E", @filled=false, @placement="E4", @row=4, @value=".">,
+ #<Cell:0x0000000130332be8 @column="E", @filled=false, @placement="E5", @row=5, @value=".">,
+ #<Cell:0x0000000130332558 @column="E", @filled=false, @placement="E6", @row=6, @value=".">]
+pry(main)> board.diagonal_for(c3)
```

Comment:
Questions about `Board#diagonal_for`... is it only expected that this method returns diagonals from **top left to bottom** right and not **bottom left to top right**?